client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -94,7 +94,11 @@ class ConversationCaterpillarComponent extends React.Component {
 
 		return (
 			<div className="conversation-caterpillar">
-				<div className="conversation-caterpillar__gravatars" onClick={ this.handleTickle }>
+				<div
+					className="conversation-caterpillar__gravatars"
+					onClick={ this.handleTickle }
+					aria-hidden="true"
```

Comment:
does this mean screen readers won't see the 🐛 as clickable?

Reply:
The linter doesn't like non-interactive elements with an `onClick`, but I think it's useful to be able to expand the caterpillar by clicking the gravatars so I wanted to keep that behaviour. On balance I thought the best option was to hide _just_ the gravatar images from assistive devices, since the "Load previous comments..." button will be available to do the expansion (it's a `button` element):

<img width="516" alt="screen shot 2018-06-04 at 11 30 35" src="https://user-images.githubusercontent.com/17325/40892392-bcc9d0f8-67ea-11e8-895b-dfb8c59c87c8.png">